intranet/utils/html.py

```diff
@@ -1,9 +1,14 @@
+from __future__ import annotations
+
 import urllib.parse
-from typing import Mapping, Optional, Tuple, Union
+from typing import TYPE_CHECKING, MutableMapping
 
 import bleach
 from bleach.css_sanitizer import CSSSanitizer
 
+if TYPE_CHECKING:
+    Attr = MutableMapping[str | tuple[str | None], str]
```

Comment:
defining a type for this seems unnecessary, I would just fix the function types without this

Reply:
You asked for it, let me know how you feel after looking at the typehints now ;)